src/encoded/static/components/viz/QuickInfoBar.js

```diff
@@ -190,64 +155,94 @@ export default class QuickInfoBar extends React.PureComponent {
     }
 
     handleActiveFilterTermClick(evt, field, term){
-        const { context } = this.props;
-        const browseBaseParams = navigate.getBrowseBaseParams();
-        const expSetFilters = QuickInfoBar.expSetFilters((context && context.filters) || null, browseBaseParams);
+        const { context, browseBaseState } = this.props;
+        const browseBaseParams = navigate.getBrowseBaseParams(browseBaseState);
+        const expSetFilters = this.memoized.expSetFilters((context && context.filters) || null, browseBaseParams);
         searchFilters.changeFilter(field, term, expSetFilters, null, false, null, browseBaseParams);
         analytics.event('QuickInfoBar', 'Unset Filter', {
             'eventLabel' : `Field: ${field}, Term: ${term}`,
             'dimension1' : analytics.getStringifiedCurrentFilters(expSetFilters)
         });
     }
 
-    renderHoverBar(){
-        const { context, browseBaseState, href, schemas } = this.props;
-        const { show, reallyShow } = this.state;
-        const browseBaseParams = navigate.getBrowseBaseParams();
-        const filters = QuickInfoBar.contextFiltersToInclude(context.filters, browseBaseParams);
-
-        if (show === 'activeFilters' || (show === false && reallyShow)) {
-            return (
-                <div className="bottom-side">
-                    <div className="crumbs-label">
-                        Filtered by
-                    </div>
-                    <ActiveFiltersBar {...{ filters, context, schemas }}
-                        onTermClick={this.handleActiveFilterTermClick}
-                        termTransformFxn={Schemas.Term.toName} fieldTransformFxn={Schemas.Field.toName} />
-                    <div className="graph-icon" onMouseEnter={null /*_.debounce(()=>{ this.setState({ show : 'mosaicCharts' }); },1000)*/}>
-                        <i className="icon icon-pie-chart fas" style={{ opacity : 0.05 }} />
-                    </div>
-                </div>
+    render(){
+        const { id, isLoadingChartData, browseBaseState, href, context, className } = this.props;
```

Comment:
Codacy found an issue: ['isLoadingChartData' is assigned a value but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6178047)